public/documentation/ios.md

```diff
@@ -0,0 +1,176 @@
+# Crosswalk Project for iOS
+
+## Introduction
+Crosswalk Project for iOS is a sub-project of Crosswalk, it aims to provide a web runtime to develop sophisticated iOS native or hybrid applications.
+
+* Extended WKWebView
+
+  Crosswalk Project for iOS is built on top of `WKWebView`, the mordern WebKit framework debuted in iOS 8.0. We extend the WKWebView to build Crosswalk extension framework within, and we are intended to bring the least intrusion to the interface to make developers use it as natural as WKWebView itself. For the detailed information you may refer to [Embedding Mode & WKWebView](ios/embedding_mode_&_native_apis).
+
```

Comment:
For these links you need to put ".html"
Example:  [Embedding Mode & WKWebView](ios/embedding_mode_&_native_apis.html).